### DISCUSSION TITLE
How to use both selection and Popup Edit in Q-Table?

### DISCUSSION THREAD

**Discussion by wendyzhang2019** [quasarframework/quasar#10247] [General - Components / Directives / etc]
We must use body scoped slot when we try to use Popup Edit. But if I enable selection feature the check-boxes disappear in first column.  I can't get it work when using body scoped slot. 
My code looks like this:
<q-table dense :columns="columns" :rows="rows" row-key="name" :selected-rows-label="selectedString" selection="multiple">
      <template v-slot:body="props" >
        <q-tr :props="props" selected>
          **_<q-td></q-td>_**
          <q-td key="userName" :props="props">{{props.row.username}}
          </q-td>
          <q-td key="nickName" :props="props">{{props.row.nickname}}
            <q-popup-edit v-model="props.row.nickname" buttons autofocus dense auto-save v-slot="scope">
              <q-input v-model="scope.value" @keyup.enter="scope.set"></q-input>
            </q-popup-edit>
          </q-td>
          <q-td key="status" :props="props">{{props.row.disabled}}</q-td>
          <q-td key="phone" :props="props">{{props.row.phone}}</q-td>
          <q-td key="operation" :props="props">{{props.row.phone}}</q-td>
        </q-tr>
      </template>
    </q-table>
If I remove the first <q-td></q-td>, the data will move one column.

**Comment by ontwikkelfabriek** [comment]
@wendyzhang2019 

For better support, please create a codepen.io that demonstrates your issue.

**Comment by Nukeap** [comment]
Works for me
```
 <template v-slot:body="props">
        <q-tr :props="props">
          <q-td key="type" :props="props">
                 <q-checkbox v-model="props.selected" />             
          </q-td>         
          <q-td key="name" :props="props">
            {{ props.row.name }}          
          </q-td>
 <q-td key="pseudo" :props="props">
            {{ props.row.pseudo }}

              <q-popup-edit v-model="props.row.pseudo" v-slot="scope">
              <q-input v-model="scope.value" dense autofocus counter @keyup.enter="scope.set" />
            </q-popup-edit>           
          </q-td>

</template>
```
<q-td> key of checkbox should be same as one of other keys of row
I think, it's wrong, but it works
